testsuite/MDAnalysisTests/util.py

```diff
@@ -262,3 +262,12 @@ def get_userid():
         return 1000
     else:
         return os.geteuid()
+
+def get_running_dask_client():
+    # the solution found here: https://stackoverflow.com/questions/59070260/dask-client-detect-local-default-cluster-already-running
+    # note: the used API is non-public and can change any time
```

Comment:
What will happen if `_get_global_client()` disappears one day? =(

Reply:
I believe there's a way to get to it via trying to connect to its default port, or simply spawning another instance of `Client` instead of using an already running one.

but I almost got rid of all the places where I'm using it, so I can try to work on completely removing it from the codebase.

Reply:
I removed this function altogether, since we're not using Client anymore.